### DISCUSSION TITLE
What does CXPatcher add for CrossOver 23.7?

### DISCUSSION THREAD

**Discussion by sofakng** [italomandara/CXPatcher#167] [General]
Please don't take this question the wrong way.  I'm only asking because I don't understand.

What does CXPatcher add for CrossOver 23.7?

My understanding was that CrossOver 23.7 already included GPTK 1.1 and MSync?

I see that "updated moltenVK" is mentioned in the release notes but CrossOver 23.7 also mentions an improved moltenVK?

Thanks for helping me understand :)

**Comment by italomandara** [comment]
Full Gstreamer support and some additional hacks this made re7 and Dying light playable.
Also UE4 hack for dxvk and faster dxvk than the stock one.
so, long story short, some extra fps for games that only run with dxvk. and extended compatibility with ue4 games that don't work with gptk.

this is a growing list of games thathave become playable with cxpatcher: https://github.com/italomandara/CXPatcher/discussions/164

**Reply by sofakng** [reply]
Thanks for the explanation!

Does CXPatcher include a different version of MoltenVK too?  What's the difference between the CrossOver 23.7 version (which mentions "nice performance boosts to MoltenVK") and the version that you include?

**Reply by italomandara** [reply]
the latest version features the stock moltenvk with a UE4 patch applied that allows a good number UE4 dx11 games to work via dxvk, usually is the latest MoltenVK included new features and bugfixes + UE4 patch.
